assets/schema_input.json

```diff
@@ -13,21 +13,28 @@
                 "errorMessage": "Sample name must be provided and cannot contain spaces",
                 "meta": ["id"]
             },
-            "fastq_1": {
+            "tempo_maf": {
```

Comment:
Looks good, any reason why we can't just use "maf" here?

Reply:
Nope, any maf should work.  I guess I had tempo on the mind while writing.